calcom/cal.com
# feat: Allow / disallow SEO indexing on booking pages

## What does this PR do?



Fixes #10465 (issue)
/claim #10465

https://github.com/calcom/cal.com/assets/54475686/5bef9099-a639-4484-818c-5d8df391bbfa

## Requirement/Documentation



- If there is a requirement document, please, share it here.
- If there is ab UI/UX design document, please, share it here.

## Type of change




## How should this be tested?



- Go to general settings of your account
- Toggle Allow SEO Indexing (should be the last option in the form) and update the profile
- Go to the your booking page for any event type, check the meta tags
- Meta tags should reflect your choice of SEO indexing

## Mandatory Tasks
